.github/workflows/ci-scrypto-builder.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     needs: tags
     uses: radixdlt/public-iac-resuable-artifacts/.github/workflows/docker-build.yml@main
     with:
-      runs_on: ubuntu-latest
+      runs_on: ubuntu-latest-16-cores
       image_registry: "docker.io"
       image_organization: "radixdlt"
       image_name: "simulator"
```

Comment:
The image name is wrong -- also this appears to publish to docker hub - is this intended?

Reply:
nope. it should not published. fixing